nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/resources/ITDevHotReloadTest/testHotReloadUpdateSchema/schema/core-types.xsd

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:complexType name="content">
+    <xs:sequence>
+      <xs:element name="encoding" type="xs:string" />
+      <xs:element name="mime-type" type="xs:string" />
+      <xs:element name="data" type="xs:base64Binary" />
+      <xs:element name="name" type="xs:string" />
+      <xs:element name="length" type="xs:long" />
+      <xs:element name="digest" type="xs:string" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:complexType name="externalcontent">
+    <xs:sequence>
+      <xs:element name="encoding" type="xs:string" />
+      <xs:element name="mime-type" type="xs:string" />
+      <xs:element name="uri" type="xs:string" />
+      <xs:element name="name" type="xs:string" />
+      <xs:element name="length" type="xs:long" />
+      <xs:element name="digest" type="xs:string" />
+    </xs:sequence>
+  </xs:complexType>
+
+</xs:schema>
```

Comment:
Not familiar with hot reload tests but imagine you cannot rely on the `core-types` registered by nuxeo-core?

Reply:
yes when i removed the import on `core-type` i had an error on the xsd indicating that the `content` type is unknown